ext/devcontainer/devcontainer-feature.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@aaronpowell - Can you do some research on how we'd publish this to the index?  @danieljurek can make it real

Reply:
To publish to GitHub Packages is quite easy, I've got a few different GitHub Action workflows that do that. https://github.com/newpatiente2e/Contoso-New-Patient-App/blob/main/.github/workflows/create-dev-image.yaml and https://github.com/aaronpowell/httpstatus/blob/main/.github/workflows/main.yaml both do it.

Reply:
I've added a new workflow to the repo that uses https://github.com/devcontainers/action to publish the feature.